### DISCUSSION TITLE
Which is the best web server provider to host Modoboa

### DISCUSSION THREAD

**Discussion by shanmugad** [modoboa/modoboa#2722] [General]
Hi everyone, 

I am looking for best Cloud services platform to host Modoboa for unlimited mailing without a restriction.

Platform and services like AWS EC2, Oracle Cloud Instance, Alibaba Cloud Instance, Azure Instance, Digital ocean, etc restrict mail services, limit number of mail can be sent or block  SMTP port. 

Is there any services provider who do not block or restrict email service

**Comment by MrGeneration** [comment]
Most likely any hosting provider will step in at a certain amount of traffic you cause.
To be honest, your question sounds fairly sketchy and like something a spammer would ask. :-)

Also "high number of mails" depends on the person that looks at the topic. I sent bursts of 60k mails per day.
When we're talking about those ratios you'll have much more trouble than just "picking the right" provider.

**Reply by shanmugad** [reply]
@MrGeneration I am not looking for spam emails. I am looking for something like 10K - 15K mail per day for my business. But the web platform which i mention above they do not allow more than 100 - 1200 mails per day.

**Reply by MrGeneration** [reply]
Hetzner does require you to pay your first bill and then to contact support for unblocking port 25 outbound.
Apart from that I couldn't find any real hard limits for selfhosted solutions neither via VPS nor dedicated servers that fit your criteria.

Yes, most providers will want you to contact their support first. That's not a bad thing but protects from automated spammers.

**Reply by Spitfireap** [reply]
> @MrGeneration I am not looking for spam emails. I am looking for something like 10K - 15K mail per day for my business. But the web platform which i mention above they do not allow more than 100 - 1200 mails per day.

For this kind of amount, I would not use something like modoboa. Even with all security practices, google can spam your domain as spam. And even without this, if this is business oriented, you would need a high uptime. This would require you to have multiple SMTP and potentially IMAP servers. This is not implemented by default on Modoboa...
Perhaps a transactional email service would suits you best ?

**Reply by shanmugad** [reply]
@Spitfireap 

What is the best & cheap transactional email service? 

and what is the best transactional email open source software like  Modoboa to build SMTP server

**Reply by shanmugad** [reply]
@MrGeneration So Modoboa do not have mail per day limit? ones they unblocked SMTP port?

**Reply by MrGeneration** [reply]
> So Modoboa do not have mail per day limit? ones they unblocked SMTP port?

No offense but if you have to ask this question you're probably wrong here.
Modoba is not a service provider but a selfhosted email server solution.

**Reply by shanmugad** [reply]
@MrGeneration sorry, it typo error. i mean to ask "So, Hetzner do not have mail per day limit? ones they unblocked SMTP port?"

**Comment by imbraintl** [comment]
Hey, Modoboa is okay, for server provider with unlimited traffic all those that you have mentioned above can do what you want. I have worked closely with all of them.
You will have to put in a request for you to be unlocked so that you are able to send that volume of data. But you will have to prove that you are not a spammer to them. Their are some documentation you will have to provide them also make sure your business is registered, before they can lift the traffic limit, you will have to have been hitting your threshold limit for a period of time.
